### DISCUSSION TITLE
A list of companies which use WebdriverIO

### DISCUSSION THREAD

**Discussion by monolithed** [webdriverio/webdriverio#6350] [Show and tell]
It would be nice to add a list of companies which use WebdriverIO!

**Comment by christian-bromann** [comment]
Absolutely! I already tweeted out to ping me if anyone uses WebdriverIO but there haven't been enough companies to show off. I will try it again. Haha this is a perfect coincident as this is the 1000th thread.

https://twitter.com/webdriverio/status/685501499431120896

# For everyone who comes across this thread: feel free to write down a small testimonial (one or two sentences) with your name and the company you are working for. I would be happy to put that on the new website!

**Comment by monolithed** [comment]
Email service at [Mail.Ru](https://en.wikipedia.org/wiki/Mail.Ru)
_After a long research of modern frameworks for integration testing, we chose the WebdriverIO. At the moment, it meets all our criteria! (Frontend team at Mail.Ru)_

**Reply by kapilbisht94** [reply]
@monolithed I'm also looking to use email service of outlook in out project through webdriverIO. Could you please help in this?

**Comment by mroien** [comment]
We use WebdriverIO at http://www.tapqa.com/ for a client Cox Media Group. Love it and @christian-bromann is a great help ALL the time.

**Comment by tikohatico** [comment]
WebdriverIO works with BDD at http://internetdevels.com as relayable and advanced automation tool.
(QA department)

**Comment by dedmau5** [comment]
Been using WebdriverIO since may of 2015 at Thomas Cook Northern Europe (http://www.thomascook.se). It is so easy to get things going and it is so reliable compared to several other frameworks out there. The api is super well organized and easy to find stuff in, and if you get stuck, the Gitter is awesome. We use WebdriverIO mainly for system regression testing on our internal and external web apps. Can't endorse this enough!

**Comment by pazone** [comment]
[Yandex](https://www.yandex.com)  is using wdio because of flexible plugin system and simple integration with other testing tools.

**Comment by anarwal** [comment]
WebdriverIO is a powerful and advanced automation tool, which has helped us with ease of good and faster execution in moving from traditional selenium webdriver. We at Axway ( https://www.axway.com/en ) are using it for our UI Automation Testing,

**Comment by georgecrawford** [comment]
We're just starting to use Webdriverio at the [Financial Times](http://ft.com), in particular to test the FT's [web app](http://app.ft.com). This presents particular challenges due to the interactive nature of the app, and its very complex single-page JS environment. The promises of Webdriverio v4 have enticed us to convert our large suite of legacy [Thucydides](http://www.thucydides.info) tests, written in a flaky and cumbersome Java codebase we all hate, to a much cleaner, terser pseudo-synchronous JS style with _far fewer_ lines of code, fewer hacks and greater stability.

Webdriverio is a comprehensive, well documented project with great coverage of the Selenium/Webdriver/Appium specs, as well as loads of very useful helper abstractions. @christian-bromann has been amazing to work with, providing fantastic support and encouraging a helpful community in general.

**Comment by dhoulker** [comment]
At [Stage Two](https://wearestagetwo.com) we use Webdriver.IO to automate as many of our tests as possible. The framework is great to work with, as is the support provided. 

The thing we've found most valuable is that with the [Cucumber boilerplate](https://github.com/webdriverio/cucumber-boilerplate/) non technical members of the test team can write tests.

WebdriverIO is a great project which we are happy to endorse.

**Comment by kencheeto** [comment]
We use webdriver.io in product engineering at [Zendesk](https://www.zendesk.com/).

**Comment by bonjonbovi87** [comment]
Here at Nokia (formally Alcatel-Lucent), we execute tens of thousands of tests daily powered by WebdriverIO.  It was chosen based on its flexibility with other tools and ease of use, allowing us to build the framework we have today.

**Comment by kevinmcdonnell** [comment]
We use WebdriverIO at [Learnosity](https://www.learnosity.com)

```
At Learnosity we use WebdriverIO to execute hundreds of UI tests as part of our CI pipeline.

The framework is easy to use, and has proven to be extensive, flexible and reliable.

The active community behind the project makes it easy to overcome any obstacles which might be encountered.
```

**Comment by jaroslav-kubicek** [comment]
It's also used in [Socialbakers](http://www.socialbakers.com/)

**Comment by xeladotbe** [comment]
We use WebdriverIO since november 2015 in order to support our 
QA department with automated frontend tests at [1&1](http://www.1and1.com).

**Comment by pimterry** [comment]
[Build Focus](http://www.buildfocus.io) uses WebdriverIO for all its system testing. It's fantastic to use, far more reliable than the other JS options, and lets you write genuinely readable tests with minimum mess.

**Comment by sajjadhossain** [comment]
THIS IS SIMLY THE FUTURE OF TESTING. EASY, OUT THE BOX, MADE BY COMMITTED DEVELOPERS AND TESTING EVANGELISTS!!!!

I am shouting this!

**Comment by lukluk** [comment]
we at loket.com willing use that too

2016-01-26 10:10 GMT+07:00 Sajjad Hossain notifications@github.com:

> THIS IS SIMLY THE FUTURE OF TESTING. EASY, OUT THE BOX, MADE BY COMMITTED
> DEVELOPERS AND TESTING EVANGELISTS!!!!
> 
> I am shouting this!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/webdriverio/webdriverio/issues/1000#issuecomment-174793528
> .

## 

Lukluk Luhuring S. S.Kom, MT
Software Engineer / Seminar Speaker

**Comment by patthiel** [comment]
We've just begun using WebdriverIO at [Urban Outfitters](http://www.urbanoutfitters.com/) and [Anthropologie](http://www.anthropologie.com) and are excited about building our integration testing framework using this great tool.

**Comment by bondarum** [comment]
I use it at <a href="http://www.instapage.com">Instapage</a> Testing our application

**Comment by dcypherthis** [comment]
We're using WebdriverIO at [Emma](http:/myemma.com) for our new JS-based integration testing framework.

**Comment by jonyet** [comment]
Some of us at [Moxie](http://moxieusa.com/) have started to migrate to the gulp-webdriver framework for testing/continuous integration, and it's been very exciting. We already use gulp and jenkins for most of our build deployment for both internal and client applications, so it's nice to see that we will be able to easily take existing selenium experience and apply it to our preferred infrastructure

**Comment by Tyriont** [comment]
We've integrated WebdriverIO into our automation testing at [1stdibs.com](http://www.1stdibs.com) which has been incredibly useful in creating clean, easy-to-read tests which are reliable and easy to maintain.  Easy to integrate with other testing tools we use, it clearly won out for us when we began converting an old Java-based testing library into JS.  The move from v2 to v3 only made things better and we're very excited for the future with v4.

**Comment by jordanQA** [comment]
[Bambu by Sprout Social](https://getbambu.com/) has found our WebdriverIO and Cucumber integration to be highly effective and reliable. The simplicity, documentation, and support from the community has allowed me to build up an automation suite without _any_ prior programming experience. Seriously. Zero programming experience. It's wonderful. Very excited for the future of WebdriverIO. - Jordan Cumming, QA Analyst @ Sprout Social.

**Comment by narainsagar** [comment]
I am working as an Application Developer at [Recurship](http://recurship.com/) and We're using WebdriverIO since Dec 2015. Currently, we've used this in our automation codebase (i.e., web crawling) and We found this very clean, reliable and easy to integrate. I'm sure we'll use this for different purposes (testing, automation, etc) in future as well.

Cheers,
Narain S.

**Comment by xbill82** [comment]
We're using WebdriverIO to test our back office at [Kuzzle.io](http://kuzzle.io/)

**Comment by christian-bromann** [comment]
Thanks for all your contributions!
Much :heart: !!!

**Comment by BrendanJMurray** [comment]
Ever since implementing WebdriverIO's visual regression service at [niche.com](url) we have been deploying with unprecedented confidence. These tests are a strong safety net that cover any potential gaps in the rest of our automated test suites.

**Comment by Shahunks** [comment]
we have implemented wdio in [zapty](https://zapty.com/)

**Comment by michaelBurko** [comment]
We're using WebdriverIO (with Cucumber integration) at [Reserve](https://reserve.com) for our new JS-based integration testing framework.

**Comment by Vincento89** [comment]
Hi everybody,
I am a great enthusiast of the WebdriverIO community. I know this is not the fully appropriate channel to ask things from you all but I have one specific problem via in the planning phase using webdriverio. How can I get in contact somehow one of you? For example email, chat or something.

**Comment by christian-bromann** [comment]
> I know this is not the fully appropriate channel to ask things

Nope it is not

> How can I get in contact somehow one of you?

You probably have seen our [![Gitter](https://badges.gitter.im/webdriverio/webdriverio.svg)](https://gitter.im/webdriverio/webdriverio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

**Comment by iamdanchiv** [comment]
Give a QA Engineer a web-automation framework and he might automate some tests. Give him [WebdriverIO](http://webdriver.io) and he will build a full-fledged, robust automation harness in a matter of days. All jokes aside, WebdriverIO was a blessing in disguise for us @[Avira](https://www.avira.com/). I'll never look back to other JS-based automation frameworks! ~[Dan Chivescu](https://www.instagram.com/iamdanchiv/), QA Lead @Avira

**Comment by CUgNhgkf** [comment]
We started using WebdriverIO at [Affirm](https://www.affirm.com/) at the beginning of 2018. I recently wrote about our initial experience with it for Affirm's [tech blog](https://tech.affirm.com/end-to-end-testing-with-webdriverio-ba33847230ad). We've really enjoyed working with it.

**Comment by msohaib980** [comment]
Just started with webdriverio last week. So far it great. Testing more and probably would migrate from webdriver java binding.

**Comment by pmerwin** [comment]
cbtnuggets.com We've been on wdio for about 3 years, now on the latest v4 release , and we are currently in beta with vs 5... we love it and are ramping up even more day to day.

**Comment by smarigowda** [comment]
Using Webdriver IO for the past 2 years. Excellent product, keep it up.

**Comment by tzero86** [comment]
I' been using WebDriverIO since early 2018 at Globant and Deloitte companies in different web projects. We've been upgrading to v5 for the last couple of weeks. Awesome tool, keep it up!

**Comment by kandarp1691** [comment]
Hello. Been using WebdriverIO since a year at [Spectrum](https://www.spectrum.com/). Amazing framework and amazing support community. Keep it up guys :)

**Comment by larryg01** [comment]
The framework of choice at Oxford University Press, I have been using webdriverio since 2016 and it has made life so much easier, especially now with v5, hats off to @christian-bromann and his crew who maintain and continualy support it, keep up the good work guys.

**Comment by sandikumar3** [comment]
Why to Choose WebDriver IO for mobile automation ? If Yes, give valid reasons with examples

**Comment by christian-bromann** [comment]
It's good to hear where WebdriverIO is used in the while. Please keep sharing!

**Comment by jonyet** [comment]
since you're asking - wdio is being leveraged happily at both _CNN_ and _Huge Inc_ for a some years now :)
